### ISSUE TITLE
Need for corpus argument

### ISSUE THREAD

**Issue by MansMeg** [raholm/tomer#18]
I do not think we need corpus here:
```
evaluate_left_to_right <- function(corpus, state, n_topics, alpha, beta, n_particles, resampling) {
```

state$type is essentially a vector that contain everything that is needed? Why do we need corpus?

**Comment by raholm** [comment]
Jag är förmodligen förvirrad, som för det mesta när det kommer till statistik, men så tanken är att evaluera korpuset som `state` består av? Inte ett annat korpus (testset),  `corpus`? För jag tolkar `state` som en del av modellen vi vill evaluera.

**Comment by MansMeg** [comment]
Du har rätt! Dock vill vi nog att det skickas in i ett tidy text format bara isf med en token per rad. D.v.s en state file men utan topics.

**Comment by MansMeg** [comment]
Har funderat lite mer på detta. Egentligen bör API:et vara två argument:
1. Ett `model` argument som är antingen en statefile (eller, på sikt, en Phi-matris).
2. Ett `test_documents` som är ett tidy text format med en token per rad

Vad tror du om det?

**Comment by raholm** [comment]
Just nu görs tokens internt men det går såklart att ändra, likaså namngivningen. `test_documents ` samt `model ` är mer beskrivande så det är ett bra förslag.

**Comment by MansMeg** [comment]
Exakt. Problemet är att man vill tokenisera på samma sätt så därför bör testsetet redan vara tokeniseras. Det är ifta ett eget problem i sig.

**Comment by raholm** [comment]
Ja, det är helt klart bättre att det kommer in som tokens då det inte är algoritmens jobb. Jag gjorde bara som gör gjort för att det är enklare för mig att testa. Enkelt att fixa senare.

**Comment by MansMeg** [comment]
Super! Ingen brådska! Ville bara påtala det.
